### ISSUE TITLE
cell area correction not documented in the manual

### ISSUE THREAD

**Issue by PKullberg** [cbig/zonation-core#18]
In the latest version of Zonation (v4) it is possible to account for differences in cell size within input layers. This feature is useful for example when input data is using geodetic coordinate system. The feature has not yet been documented in the manual, but is fully functional. 

To use cell size correction add following line to the settings file (.dat):
`cell area layer = path_to_a_raster_layer_indicating _cell_size`

**Comment by jlehtoma** [comment]
Would it be useful to have the undocumented features described in [the wiki](https://github.com/cbig/zonation-core/wiki) of this repo? Or is this more trouble than useful?

**Comment by PKullberg** [comment]
You are right, wiki would the best place for describing new features. I simply created a page for this issue to the wiki and moved the content there. It is a bit lonely there at the moment, but maybe in the future we could put a little bit more effort to the the wiki.
